In [1]:
import pandas as pd
import numpy as np

In [2]:
df = pd.read_csv("./data/processed_data.csv", encoding="utf-8")

In [3]:
df.head()

user_id       order_id        time  \
0  BDS49BDS  XBH011609/006  2016-09-05   
1  BDS49BDS  XBH011609/005  2016-09-05   
2  BDS49BDS  XBH011609/005  2016-09-05   
3  BDS49BDS  XBH011609/005  2016-09-05   
4  BDS49BDS  XBH011609/004  2016-09-05   

                                description       product_id  \
0  Xuất bán T09/06- DYKEH ngày ETD 5/9/2016  LTSCORMH374SCA1   
1  Xuất bán T09/05- DYKEH ngày ETD 5/9/2016  LTSCORMH374SCA1   
2  Xuất bán T09/05- DYKEH ngày ETD 5/9/2016  LTSCORCH374SCA1   
3  Xuất bán T09/05- DYKEH ngày ETD 5/9/2016  LTSCOR1H374SCA1   
4  Xuất bán T09/04- DYKEH ngày ETD 5/9/2016  LTTCORVH374SCA1   

                    product_name  amount    price  
0    Corner Sofa Set_Middle_H374    68.0  1812.88  
1    Corner Sofa Set_Middle_H374    68.0  1812.88  
2    Corner Sofa Set_Corner_H374    34.0  1813.22  
3  Corner Sofa Set_ArmChair_H374    34.0  1813.22  
4     Corner Sofa Set_Table_H374    34.0  1814.24

In [4]:
df.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2415 entries, 0 to 2414
Data columns (total 8 columns):
user_id         2415 non-null object
order_id        2415 non-null object
time            2415 non-null object
description     2415 non-null object
product_id      2415 non-null object
product_name    2415 non-null object
amount          2415 non-null float64
price           2415 non-null float64
dtypes: float64(2), object(6)
memory usage: 151.1+ KB


In [6]:
df.describe(include="all").T

count unique                                               top  \
user_id       2415     45                                          AQU49LCW   
order_id      2415   1132                                     XBH011606/026   
time          2415    224                                        2017-01-21   
description   2415   1131  Xuất bán T06/26 - LT9516CLFA  ngày ETD 26/6/2016   
product_id    2415    496                                   LTCGUD1F310SPA1   
product_name  2415    495                                Gudhjem Chair_F310   
amount        2415    NaN                                               NaN   
price         2415    NaN                                               NaN   

             freq     mean      std  min    25%   50%    75%    max  
user_id       422      NaN      NaN  NaN    NaN   NaN    NaN    NaN  
order_id       35      NaN      NaN  NaN    NaN   NaN    NaN    NaN  
time           98      NaN      NaN  NaN    NaN   NaN    NaN    NaN  
description    35      NaN      NaN  NaN    NaN   NaN    NaN    NaN  
product_id     86      NaN      NaN  NaN    NaN   NaN    NaN    NaN  
product_name   86      NaN      NaN  NaN    NaN   NaN    NaN    NaN  
amount        NaN  382.737  567.262    1     14    78    720   5040  
price         NaN   5540.1  6231.51    0  637.2  2618  10080  31620

### Summary
- user
    - Total: 45
    - Most freq buyer: AQU49LCW (422 times)
    - 
- product
    - Total: 496
    - Most popular product: LTCGUD1F310SPA1 (86 times)